### ISSUE TITLE
stemcell-uploader has hardcoded vsphere in wget 

### ISSUE THREAD

**Issue by MarkDyson** [rahul-kj/concourse-vsphere#21]
Trying to run some tile pipelines with Azure-hyperv and trusty stemcells but fails each time as it trys to find them in the vsphere repo.

code below:
```    
 if [[ -z "$RESPONSE" ]]; then
      wget --show-progress `https://s3.amazonaws.com/bosh-core-stemcells/vsphere/$STEMCELL_NAME`
      else
      $PIVNET_CLI download-product-files -p $PRODUCT_SLUG -r $SC_VERSION -g "*$IAAS_TYPE*" -- 
      accept-eula
    fi    
```

**Comment by MarkDyson** [comment]
os build (i.e. trusty) is also not included in the $STEMCELL_NAME. See output from 'upload-stemcall' job:

```
wget --show-progress https://s3.amazonaws.com/bosh-core-stemcells/vsphere/bosh-stemcell-3541.34-azure-hyperv-ubuntu--go_agent.tgz
--2018-09-29 03:13:19--  https://s3.amazonaws.com/bosh-core-stemcells/vsphere/bosh-stemcell-3541.34-azure-hyperv-ubuntu--go_agent.tgz
```

**Comment by rahulkj** [comment]
I'm in the process of cleaning up that logic. Testing it and will push the changes coon. Thanks for reporting it @MarkDyson

**Comment by rahulkj** [comment]
resolved in v-0.8.6
